src/Plugin.php

```diff
@@ -198,6 +198,35 @@ public static function getBasePath() {
     return dirname(__DIR__);
   }
 
+  /**
+   * Gets the reference of the last commit.
+   *
+   * Helper function to handle asset version with wp_enqueue_*
+   * functions to bust caches.
+   *
+   * @see https://developer.wordpress.org/reference/functions/wp_enqueue_script/
+   *
+   * @return string|null
+   *   The first 8 characters of the git reference.
+   */
+  public static function getGitRef() {
```

Comment:
```suggestion
  public static function getGitCommitHash() {
```

Reply:
This feature should be moved into a separate class/file within Core Standards, so that it is properly encapsulated. For example, class `Asset`